ocpmodels/datasets/ase_datasets.py

```diff
@@ -110,6 +122,31 @@ def close_db(self):
         pass
         # This method is sometimes called by a trainer
 
+    def guess_target_metadata(self, num_samples=100):
+        metadata = {}
+
+        if num_samples < len(self):
+            metadata["targets"] = guess_property_metadata(
+                [
+                    self.get_atoms_object(self.ids[idx])
+                    for idx in np.random.choice(
+                        self.__len__(), size=(num_samples,), replace=False
```

Comment:
Nit: `self.__len__()` --> `len(self)` is more pythonic.

Reply:
Done!